### PR TITLE
docs: update prerequisite section

### PR DESCRIPTION
**Pull request by watson**
Renamed "Installation Requirements" to "Prerequisites" and attempted to make it more universal and less reliant on non-standard package managers.

### CODE REVIEW

**Comment by graphaelli** [comment]
This shouldn't be needed once #99 goes in.   That in turn was blocked but I think we can resolve all of that today.

**Comment by watson** [comment]
It should be updated to show support for Python 2 sure, but what do you mean by it's not needed?

**Comment by watson** [comment]
Ah, is the virtualenv requirement gone as well?

**Comment by graphaelli** [comment]
Right, no need to virtualenv for just standing up an environment, only python 2 or 3 stdlib is used. 
 virtualenv + python3 is required for running the apm integration tests, even under docker.

**Comment by graphaelli** [comment]
@watson good to close this one?

**Comment by watson** [comment]
Lost track of it. I still think the requirements section could be improved a bit, so I tried to address your earlier comments and rebased with master.

I wanted to get the reference to installing docker with Homebrew out and instead link to the official Docker install docs. If there was a reason why we showed the brew instructions let me know 😃 

I also added a section on how to install Python (but left the note about the Python versions)

What do you think?

File: README.md [hunk 1]
```diff
@@ -4,20 +4,30 @@ This repo contains tools for end to end (eg agent -> apm server -> elasticsearch
...
+[Installation instructions](https://www.docker.com/community-edition)
+
+### Python 3
+
+- Windows: [Installation instructions](https://www.python.org/downloads/windows/)
+- Mac (using [Homebrew](https://brew.sh/)):
+  ```sh
+  brew install python
+  ```
+- Linux
```
Comment (graphaelli) [inline README.md]: This should be something like `Debian/Ubuntu`

Comment (watson) [reply README.md]: Good point. Fixed :)

**Pull request merged**